platform/rct/Makefile

```diff
@@ -252,33 +251,37 @@ inc-y += $(RC_PLATFORM_DIR)/mk2/hal/fat_sd_stm32/fatfs
 ################################################################################
 obj-y := $(addprefix $(OUTDIR)/,$(patsubst %.s,%.o,$(patsubst %.c,%.o,$(src-y))))
 
+#
+# Our chip is actually a cortex-m4 with FPU.  We have not included the correct
+# FreeRTOS libs as well as compiler args to make it work properly with this arch
+# so for now this stays as an M3.
```

Comment:
Yeah... I know.  I was a bit shocked to learn this.  The F1 and F2 series are cortex M3, F3 and F4 are cortex M4 chips.